### PR TITLE
Add failing test fixture for RemoveJustPropertyFetchRector

### PR DESCRIPTION
**Pull request by kkmuffme**
# Failing Test for RemoveJustPropertyFetchRector

Based on https://getrector.org/demo/94b6b311-0942-448d-a585-337f7d0b38d7

### CODE REVIEW

**Comment by TomasVotruba** [comment]
This will be fixed in php-parser 5.0: https://github.com/nikic/PHP-Parser/commit/b0edd4c41111042d43bb45c6c657b2e0db367d9e

We'll have to first wait for the release and PHPStan adoption.

**Comment by samsonasik** [comment]
It resolved in latest rector/rector:dev-main with include php-parser 4.15.2, ref https://getrector.org/demo/c8c9fbdb-932b-4af6-8072-826667d6614f

You can try :

```
composer update --dev rector/rector:dev-main -W
```

**Comment by TomasVotruba** [comment]
Thank you @samsonasik :+1:

File: rules-tests/DeadCode/Rector/StmtsAwareInterface/RemoveJustPropertyFetchRector/Fixture/demo_file.php.inc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\StmtsAwareInterface\RemoveJustPropertyFetchRector\Fixture;
+
+final class DemoFile
+{
+    public function run( $arg )
+    { ?>
+        <input type="text" value="<?php echo $arg->name; ?>">
+    <?php }
```
Comment (samsonasik) [inline rules-tests/DeadCode/Rector/StmtsAwareInterface/RemoveJustPropertyFetchRector/Fixture/demo_file.php.inc:25]: new line after open tag after html based on https://getrector.org/demo/c8c9fbdb-932b-4af6-8072-826667d6614f

**Comment by samsonasik** [comment]
This issue is resolved, the next is you can use coding standard tool if you want tidify :)

**Pull request closed without merging**